src/cli/argument-parser.ts

```diff
@@ -252,6 +256,19 @@ export default class CLIArgumentParser {
         this.opts.src = this.program.args.slice(1);
     }
 
+    private async _parseScreenshotOptions (): Promise<void> {
+        if (this.opts.screenshots)
+            this.opts.screenshots = await getScreenshotOptions(this.opts.screenshots);
+        else
+            this.opts.screenshots = {};
+
+        if (!has(this.opts.screenshots, SCREENSHOT_OPTION_NAMES.pathPattern) && this.opts.screenshotPathPattern)
+            this.opts.screenshots[SCREENSHOT_OPTION_NAMES.pathPattern] = this.opts.screenshotPathPattern;
```

Comment:
```JS
this.opts.screenshots[SCREENSHOT_OPTION_NAMES.pathPattern] = get(this.opts.screenshots, SCREENSHOT_OPTION_NAMES.pathPattern) || this.opts.screenshotPathPattern
```

May be this case is better? Anyway, I've already approved the PR

Reply:
I wanted to allow overriding legacy keys with `false` and `''` values. It won't work if we will just use `get`.